hypothesis-python/tests/cover/test_stateful.py

```diff
@@ -10,6 +10,8 @@
 
 import base64
 import re
+import inspect
+import trio
```

Comment:
We'd want to make this an optional dependency, and thus put the tests in e.g. `tests/trio/test_trio_stateful.py`; as above I suggest starting with asyncio and putting those tests in `tests/cover/test_asyncio_stateful.py`.